dccrg.hpp

```diff
@@ -10265,7 +10299,55 @@ template <
 	}
 
 public:
-	/*!
+	void force_update_cell_neighborhoods(const std::vector<uint64_t> cells)
```

Comment:
Semantic comment:
This method is called *force*, but the first thing it does is check whether it actually really wants to.
So it's not really forcing, is it?

(Would it be much more complex to simply just recalculate them all, in that situation?)

Reply:
well, it skips the check of the cell being local to the current process. But true, not necessarily the best wording choice.